### ISSUE TITLE
Getting back not valid value for -webkit-box-ordinal-group property when setting order: inherit; or order: initial;

### ISSUE THREAD

**Issue by jnnkm** [postcss/autoprefixer#799]
Hi everybody,

I encountered some weird behavior of the postcss-autoprefixer plugin which might be of interest.

When setting the order property to,

```css
order: inherit; 
```
or 
```css
order: initial;
```

I get this back:

```css
// using order: inherit;
-webkit-box-ordinal-group: NaN;
-ms-flex-order: inherit;
order: inherit;
```

```css
// using order: initial;
-webkit-box-ordinal-group: NaN;
-ms-flex-order: initial;
order: initial;
```

As you can see the -webkit-box-ordinal-group value is not valid.

it works fine when setting
```css
order: 0;
```
I made a codepen example showing this issue,
you can find it here: https://codepen.io/jnnkm/pen/MppBaK

I haven't found any similar issues in this repository,
have you seen this before?

Thank you!

**Comment by ai** [comment]
Oops 😅. I will fix it on this weekend.

**Comment by ai** [comment]
Done b12960c

**Comment by ai** [comment]
Released in 6.7.7.
